### PR TITLE
Added Bounced props / Add index and focused props for Screen

### PR DESCRIPTION
**Pull request by m-tymchyk**

### CODE REVIEW

**Comment by m-tymchyk** [comment]
Could you fix a tests to not check dependency  of Example?

**Comment by satya164** [comment]
> Could you fix tests to not check dependency of Example?

The example must pass flow. Otherwise, when you use the same code in your app, it's gonna fail as well.

File: src/TabView.js [hunk 1]
```diff
@@ -189,10 +189,12 @@ export default class TabView<T: *> extends React.Component<Props<T>, State> {
             ...rest,
             panX: this.state.panX,
             offsetX: this.state.offsetX,
-            children: navigationState.routes.map(route => {
+            children: navigationState.routes.map((route, index) => {
               const scene = this._renderScene({
                 ...props,
                 route,
+                index: index,
+                focused: index === navigationState.index,
```
[the remark this answers is not in the document]
Comment (m-tymchyk) [reply src/TabView.js]: But we have it at declaration:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bc0c933415466b34d2de5790f7cd6418f676801e/types/react-native-tab-view/index.d.ts#L28

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. It's gone through a major refactor and there will be conflicts with your PR.

Please try the new version and see if it addresses your issue. If not, please open a new PR with the change.

**Pull request closed without merging**